extensions/sql-database-projects/src/dialogs/addDatabaseReferenceDialog.ts

```diff
@@ -327,11 +335,38 @@ export class AddDatabaseReferenceDialog {
 			this.databaseVariableTextbox!.value = isSystemDb ? '' : this.databaseVariableTextbox!.value;
 			this.serverNameTextbox!.value = '';
 			this.serverVariableTextbox!.value = '';
+
+			// add default values in enabled fields
+			this.setDefaultDatabaseValues();
 		} else if (this.locationDropdown?.value === constants.differentDbDifferentServer) {
 			this.databaseNameTextbox!.enabled = true;
 			this.databaseVariableTextbox!.enabled = true;
 			this.serverNameTextbox!.enabled = true;
 			this.serverVariableTextbox!.enabled = true;
+
+			// add default values in enabled fields
+			this.setDefaultDatabaseValues();
+			this.serverNameTextbox!.value = constants.otherServer;
+			this.serverVariableTextbox!.value = constants.otherSeverVariable;
+		}
+	}
+
+	private setDefaultDatabaseValues(): void {
+		switch (this.currentReferenceType) {
+			case ReferenceType.project: {
+				// TODO: add when projects support is added
```

Comment:
Can you make this into a GH issue and add the ID here?

Reply:
I'll add this in this PR after #12186 is merged